src/Wrapper.js

```diff
@@ -207,7 +207,13 @@ export default class Wrapper implements WrapperInterface {
       error('wrapper.hasClass() must be passed a string');
     }
 
-    return this.element.className.split(' ').indexOf(className) !== -1;
+    let classes = this.element.className;
```

Comment:
We could use classList instead of className. Would you be willing to ammend your PR with classList instead of className?

Reply:
Yes, of course, I will add soon.